docs/fsharp/getting-started-netcore.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Getting started with F# on .NET Core
+description: Getting started with F# on .NET Core
+keywords: .NET, .NET Core
+author: cartermp
+manager: wpickett
+ms.date: 07/01/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 615db1ec-6ef3-4de2-bae6-4586affa9771
+---
+
+# Getting started with F# on .NET Core
+
+This article covers how you can get started with using F# on .NET Core with the .NET Core 1.0 Preview 2 SDK.  It will go through building a multi-project solution with a Class Library, Console App, and xUnit test project.
+
+## Prerequisites
+
+To begin, you must install the [.NET Core 1.0 SDK Preview 2](https://www.microsoft.com/net/core).
+
+This article uses the command line and [Visual Studio Code](https://code.visualstudio.com) as the text editor.  You can use any editor you like.
+
+## Building a Simple Multi-project Solution
+
+1. Open up a Command Line/Terminal.
+2. Create a new directory named `FSNetCore`.  Open Visual Studio code or your preferred editor inside this directory. 
+3. Under `FSNetCore`, create `src` and `test` directories.
+4. Under `FSNetCore`, create a new file called `global.json`.  It should have this as its contents:
+
+```json
+{
+    "projects":[ "src", "test" ]
+}
+```
+
+Your solutions structure should now look like this:
+
+```
+FSNetCore/
+|---src/
+|---test/
+|---global.json
+```
+
+### Writing a Class library
+
+1. Create a `Library` folder under `FSNetCore/src`.
+2. In the command line, execute `dotnet new -l F#` in `FSNetCore/src/Library`.
+3. Remove the `NuGet.Config` file.
+4. Rename `Program.fs` to `Lib.fs`.
+5. Open the `project.json` file and remove the `emitEntryPoint` entry from `buildOptions`.
+6. Under `buildOptions/compile/includeFiles`, replace `Program.fs` with `Lib.fs`.
+7. Remove the global `dependencies` section.
+8. Under `frameworks`, change `netcoreapp1.0` to `netstandard1.6`.
+9. Under `frameworks/netstandard1.6`, remove the `imports` section.
+10. Under `frameworks/netstandard1.6/dependencies`, replace the `Microsoft.NETCore.App` package with `"NETStandard.Library":"1.6.0"`.  Add `"Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629"` and `"Newtonsoft.Json": "9.0.1-beta1"`.
```

Comment:
@enricosada Even though we control these packages, I prefer the fixed version number here.  The splat string is something to generally avoid so it just aligns with our stated best practices for packages in general.

@svick I suppose it would be a simpler doc if it were to just say, "copy and paste this `project.json`.  Hmmm.  I'll think about that.